src/globus_cli/commands/bookmark/list.py

```diff
@@ -2,7 +2,24 @@
 
 from globus_cli.login_manager import LoginManager
 from globus_cli.parsing import command
-from globus_cli.termio import formatted_print
+from globus_cli.termio import Field, display, formatters
+
+
+class EndpointIdToNameFormatter(formatters.StrFormatter):
+    def __init__(self, client: globus_sdk.TransferClient) -> None:
+        self.client = client
+
+    def render(self, value: str) -> str:
+        from globus_cli.services.transfer import display_name_or_cname
```

Comment:
All imports from the `globus_cli.services` subpackage are deferred as a performance optimization.
I do it by policy because it's easier to be consistent than to reason about which _particular_ imports cause perf regressions.

In the case of `globus_cli.services.transfer`, we have a custom subclass of `globus_sdk.TransferClient`. Subclassing forces the `globus_sdk` lazy importer to pull in `TransferClient` -- this turns into a dependency chain which pulls in `requests` and causes a big performance hit for tab completions and `--help`.

Reply:
Excellent, that answers my question. Thanks!